tests/ui/blacklisted_name.stderr

```diff
@@ -1,88 +1,16 @@
-error: use of a blacklisted/placeholder name `foo`
-  --> $DIR/blacklisted_name.rs:11:9
+error: lint `clippy::blacklisted_name` has been renamed to `clippy::disallowed_name`
+  --> $DIR/blacklisted_name.rs:9:9
    |
-LL | fn test(foo: ()) {}
-   |         ^^^
+LL | #![warn(clippy::blacklisted_name)]
+   |         ^^^^^^^^^^^^^^^^^^^^^^^^ help: use the new name: `clippy::disallowed_name`
```

Comment:
Please include the rename tests in `tests/ui/rename.rs`

Reply:
Okay, shall I remove `tests/ui/blacklisted_name ` then?

Reply:
Yes, after you have one `#[warn(clippy::blacklisted_name)]` in the rename test file you can remove this.